core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy

```diff
@@ -57,4 +57,14 @@ class BintrayConfiguration {
         }
     }
 
+    private void deriveDefaultsFromProject(Project project) {
+        if (extension.versionAttributes.isEmpty()) {
+            def gradlePluginPropertyFinder = new GradlePluginPropertyFinder(project)
+            String bestPluginId = gradlePluginPropertyFinder.findBestGradlePluginId()
+            if (bestPluginId != null) {
+                extension.versionAttributes << ['gradle-plugins': "$bestPluginId:$extension.groupId:$extension.artifactId"]
+                println "Using plugin identifier '" + extension.versionAttributes.get('gradle-plugins') + "' for gradle portal."
```

Comment:
As we pick from possible several ids I think yes
Am 29.06.2015 14:28 schrieb "Xavi Rigau" <notifications@github.com>:

> In
> core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy
> <https://github.com/novoda/bintray-release/pull/59#discussion_r33458217>:
>
> > @@ -57,4 +57,14 @@ class BintrayConfiguration {
> >          }
> >      }
> >
> > +    private void deriveDefaultsFromProject(Project project) {
> > +        if (extension.versionAttributes.isEmpty()) {
> > +            def gradlePluginPropertyFinder = new GradlePluginPropertyFinder(project)
> > +            String bestPluginId = gradlePluginPropertyFinder.findBestGradlePluginId()
> > +            if (bestPluginId != null) {
> > +                extension.versionAttributes << ['gradle-plugins': "$bestPluginId:$extension.groupId:$extension.artifactId"]
> > +                println "Using plugin identifier '" + extension.versionAttributes.get('gradle-plugins') + "' for gradle portal."
>
> is this log needed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/bintray-release/pull/59/files#r33458217>.
>